src/main/app/claims/models/claim.ts

```diff
@@ -238,13 +238,27 @@ export class Claim {
     if (this.eligibleForCCJAfterBreachedSettlement) {
       statuses.push({ status: ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_BREACHED_SETTLEMENT })
     }
+
     if (!this.moneyReceivedOn || (!this.moneyReceivedOn && !this.countyCourtJudgmentRequestedAt)) {
-      statuses.push({ status: ClaimStatus.PAID_IN_FULL_ELIGIBLE })
+      if (this.isOfferSubmitted() || this.isResponseSubmitted() || (this.response && (this.response as FullAdmissionResponse).paymentIntention.paymentOption !== PaymentOption.IMMEDIATELY)
+        && !(this.admissionPayImmediatelyPastPaymentDate) && !(this.isSettlementReachedThroughAdmission())
+      || !this.response) {
+        if (!(this.countyCourtJudgmentRequestedAt &&
+          (this.hasClaimantSuggestedAlternativePlanWithCCJ() ||
+            this.hasClaimantAcceptedAdmissionWithCCJ()))) {
+          statuses.push({ status: ClaimStatus.PAID_IN_FULL_ELIGIBLE })
+        }
+      }
     }
 
     return statuses
   }
 
+  get admissionPayImmediatelyPastPaymentDate (): boolean {
```

Comment:
Please rename it to `isFullAdmissionPayImmediatelyPastPaymentDate()`

Reply:
I have copied this method from Surjeet's branch, so I'm going to keep the name the same to avoid merge conflicts.